templates/admin-simple-product-subscription-fields.html.php

```diff
@@ -15,15 +15,23 @@
         display: none !important;
     }
 </style>
-
 <div class="options_group vindi-subscription_pricing show_if_subscription show_if_variable-subscription">
 
 <?php
+
+    if(preg_match('/variable-subscription/', $product_type)) {
```

Comment:
```suggestion
    if (preg_match('/variable-subscription/', $product_type)) {
```

Podemos escrever assim?

Reply:
Sim